compiler/test/dotty/tools/vulpix/ParallelTesting.scala

```diff
@@ -839,6 +850,24 @@ trait ParallelTesting extends RunnerOrchestration { self =>
     }
   }
 
+  def diffMessage(sourceTitle: String, outputLines: Seq[String], checkLines: Seq[String]): Option[String] = {
+    def linesMatch =
+      outputLines
+        .zip(checkLines)
+        .forall { case (x, y) => x == y }
```

Comment:
To avoid intermediate collection, could be:
```scala
(outputLines, checkLines).zipped.forall(_ == _)
```

Reply:
Updated